src/log.py

```diff
@@ -0,0 +1,55 @@
+"""
+Data Log
+
+By: Norma Chang
+"""
+import json
+import time
+
+def log(data, data_type):
+    """
+    Logs dictionary data from leader
+    """
+    if data_type == 'leader':
+        leader_dict = {
+            'Time' : time.time(), 
+            'Input' : data._distance,
+            'Speed' : data._speed,
+        }
+        write_to_file(leader_dict, data_type)
+    elif data_type == 'follower':
+        follower_dict = {
+            'Time' : time.time(), 
+            'Distance' : data._distance,
+            'Speed' : data._speed,
+            'Turn Angle' : data._turn_angle,
+            'Yaw' : data._yaw,
+        }
+        write_to_file(follower_dict, data_type)
+    else:
+        pass
+
+def write_to_file(data, file_name):
+    """
+    Write directionary data to text file.
+    """
+    with open(file_name + '.log', 'a') as outfile:
+        json.dump(data, outfile)
+        outfile.write('\n')
+
+def array_dict(file_name):
```

Comment:
The whole function body is not indented correctly. This has 2 spaces per indentation while the rest of the project has 4 spaces.

Reply:
Did you ever run this? Even the rest of the file has 4 spaces.

Reply:
fixed file spacing.